README.md

```diff
@@ -42,157 +42,161 @@ Optionally include the following parameters when running the cli:
 
-`$ phoenix-cli -u dev get-transaction -m 5iLqmcg8vifdnnw6wEpVtQxFE4Few5uiceDWzi3jvzH8 -s 4AA2WecMCJ3AsXJw2onLnihdpeCYw7tGDVHexMY37civAdyUnk8uP2aiiz64LQrLMBt4B62csYiFKQGWN33PuKyJ`
+`$ phoenix-cli -u main get-transaction 4DoNfFBfF7UokCC2FQzriy7yHK6DY6NVdYpuekQ5pRgg 4gw6UDWsDCWrh2eqYxvVzbVyywfPVo24V2qMTSVGJJAdxvv9Tx4pBrqE1cLTgomP2QkZ7wigbjoN3GpibhJY8PFV`
 ```
-market: 5iLqmcg8vifdnnw6wEpVtQxFE4Few5uiceDWzi3jvzH8, event_type: Place, timestamp: 1671652667, signature: 4AA2WecMCJ3AsXJw2onLnihdpeCYw7tGDVHexMY37civAdyUnk8uP2aiiz64LQrLMBt4B62csYiFKQGWN33PuKyJ, slot: 183671273, sequence_number: 1488049, event_index: 0, maker: mkrc4jMLEPRoKLUnNL7Ctnwb7uJykbwiYvFjB4sw9Z9, taker: , price: 11.895, side: Bid, quantity: 168.137
-market: 5iLqmcg8vifdnnw6wEpVtQxFE4Few5uiceDWzi3jvzH8, event_type: Place, timestamp: 1671652667, signature: 4AA2WecMCJ3AsXJw2onLnihdpeCYw7tGDVHexMY37civAdyUnk8uP2aiiz64LQrLMBt4B62csYiFKQGWN33PuKyJ, slot: 183671273, sequence_number: 1488050, event_index: 0, maker: mkrc4jMLEPRoKLUnNL7Ctnwb7uJykbwiYvFjB4sw9Z9, taker: , price: 12.21, side: Ask, quantity: 409.5
-market: 5iLqmcg8vifdnnw6wEpVtQxFE4Few5uiceDWzi3jvzH8, event_type: Place, timestamp: 1671652667, signature: 4AA2WecMCJ3AsXJw2onLnihdpeCYw7tGDVHexMY37civAdyUnk8uP2aiiz64LQrLMBt4B62csYiFKQGWN33PuKyJ, slot: 183671273, sequence_number: 1488051, event_index: 0, maker: mkrc4jMLEPRoKLUnNL7Ctnwb7uJykbwiYvFjB4sw9Z9, taker: , price: 11.71, side: Bid, quantity: 426.985
+
 ```
 
 ### get-market-status
 Returns the status of a given market. Markets can be in the following states: Active, PostOnly, Paused, Closed, Uninitialized, Tombstoned.
 
-`$ phoenix-cli -u dev get-market-status -m 5iLqmcg8vifdnnw6wEpVtQxFE4Few5uiceDWzi3jvzH8`
+`$ phoenix-cli -u main get-market-status 4DoNfFBfF7UokCC2FQzriy7yHK6DY6NVdYpuekQ5pRgg`
 ```
 Market status: Active
 ```
 
 ### get-seat-info
 Returns the status and address of a trader's seat. By default, returns the payer's seat info. Seats can be in the following states: Approved, NotApproved, Retired
 
-`$ phoenix-cli -u dev get-seat-info -m 5iLqmcg8vifdnnw6wEpVtQxFE4Few5uiceDWzi3jvzH8 -t mkrc4jMLEPRoKLUnNL7Ctnwb7uJykbwiYvFjB4sw9Z9`
+`$ phoenix-cli -u main get-seat-info -m 4DoNfFBfF7UokCC2FQzriy7yHK6DY6NVdYpuekQ5pRgg -t 3HBWHuyxWv4uN8U8SeukocrWPfLZJqrtj9DgDHsGo2HR`
 ```
-Seat address: mg6uXraBkvi7ccbnKvJoXgyjUVDBSqcxDJysPqiMYau
+Seat address: GGyZqgoqnKsvMTsmSSkTrDjtdSFUsEoioKz9Yr2vEnZa
 Seat status: Approved
 ```
 
 ### get-open-orders
 Returns all open orders on a given market for a trader. By default, returns the payer's open orders. Returns the side, orderID, price in ticks, price, and size for each order. 
 
-`$ phoenix-cli -u dev get-open-orders -m 5iLqmcg8vifdnnw6wEpVtQxFE4Few5uiceDWzi3jvzH8 -t mkrc4jMLEPRoKLUnNL7Ctnwb7uJykbwiYvFjB4sw9Z9`
+`$ phoenix-cli -u main get-open-orders 4DoNfFBfF7UokCC2FQzriy7yHK6DY6NVdYpuekQ5pRgg -t mkrc4jMLEPRoKLUnNL7Ctnwb7uJykbwiYvFjB4sw9Z9`
```

Comment:
this one needs `-m`

and also get Trader not found for this example

can either substitute a real trader, or just have the example market be the BASE/QUOTE market

Reply:
why do we want -m on this one